app/layouts/Dashboard/index.js

```diff
@@ -1,17 +1,27 @@
 import React from 'react';
 import Navbar from 'Components/Navbar';
 import Sidebar from 'Components/Sidebar';
+import classNames from 'classnames';
+
+import { useRouter } from 'next/router';
 
 import './style.scss';
 
 const Dashboard = ({ children, data }) => {
   if (!data) return null;
 
+  const isEmbedAvailable = useRouter().query.hasOwnProperty('embed');
```

Comment:
The hooks should not be conditionally rendered. This should be the first line inside the function.

rename to `isEmbed`.